### ISSUE TITLE
Sound doesn't play when boss is discovered

### ISSUE THREAD

**Issue by KyrosKrane** [fstenstrom/world-boss-timers#28]
New error that first started appearing after patch 8.2:

```Date: 2019-06-26 13:06:36
ID: 5
Error occured in: Global
Count: 120
Message: ..\AddOns\WorldBossTimers\Util.lua line 122:
   PlaySoundFile Error - Invalid fileDataID for sound.
Debug:
   [C]: ?
   [C]: PlaySoundFile()
   WorldBossTimers\Util.lua:122: PlaySoundAlert()
   WorldBossTimers\WorldBossTimers.lua:289:
      WorldBossTimers\WorldBossTimers.lua:281
   WorldBossTimers\WorldBossTimers.lua:315:
      WorldBossTimers\WorldBossTimers.lua:302
Locals:

AddOns:
  (trimmed for readability)
```

This happened when Galleon spawned and was instantly engaged in combat. The timer itself did update correctly.

**Comment by calebwidmer** [comment]
thanks for reporting, I came here to report the same issue (ditto!)

**Comment by calebwidmer** [comment]
Love the addon TYVM!
I use it alot and promote it when i can!
**if it helps npc scan has same/similar issue:**
https://www.curseforge.com/wow/addons/npcscan/issues/274
**contents of referenced above URL:**

Start at line 108 in alerts.lua
Replace the sound lines with this:

LibSharedMedia:Register("sound", "NPCScan Killed", 567394) -- Sound\Interface\RaidBossWarning.ogg
LibSharedMedia:Register("sound", "NPCScan Chimes", 1489461) -- Sound\Interface\UI_Legendary_Item_Toast.ogg
LibSharedMedia:Register("sound", "NPCScan Gruntling Horn", 598196) -- Sound\Events\gruntling_horn_bb.ogg
LibSharedMedia:Register("sound", "NPCScan Ogre War Drums", 567275) -- Sound\Event Sounds\Event_wardrum_ogre.ogg
LibSharedMedia:Register("sound", "NPCScan Scourge Horn", 567386) -- Sound\Events\scourge_horn.ogg

**Comment by fstenstrom** [comment]
Thanks for reporting, fixed in 3390d9b
